root/taglookup/Index.js

```diff
@@ -10,19 +10,24 @@
 import React from 'react';
 
 import Layout from '../layout';
+import {WEB_SERVER} from '../static/scripts/common/DBDefs';
 
 import TagLookupForm from './Form';
 import type {TagLookupPropsT} from './types';
 import TagLookupNagSection from './Nag';
 
-const TagLookup = (props: TagLookupPropsT) => (
-  <Layout fullWidth title={l('Tag Lookup')}>
-    <div className="content">
-      <h1>{l('Tag Lookup')}</h1>
-      {props.nag ? <TagLookupNagSection /> : null}
-      <TagLookupForm form={props.form} />
-    </div>
-  </Layout>
-);
+const TagLookup = (props: TagLookupPropsT) => {
+  const musicbrainzRegex = /^(beta\.)?musicbrainz\.org$/;
+  const isMusicBrainzServer = WEB_SERVER.match(musicbrainzRegex);
+  return (
+    <Layout fullWidth title={l('Tag Lookup')}>
+      <div className="content">
+        <h1>{l('Tag Lookup')}</h1>
+        {props.nag && isMusicBrainzServer ? <TagLookupNagSection /> : null}
```

Comment:
Hmm, since `props.nag` is calculated on the server, perhaps we should just change the calculation there instead of adding additional logic here.

i.e. changing `sub nag_check` in lib/MusicBrainz/Server/Controller/TagLookup.pm to `return 0` early if `DBDefs->WEB_SERVER` matches? Then the component here wouldn't need modification at all.

Reply:
That seems like a good idea. Changed.